src/rules/export.js

```diff
@@ -24,6 +24,19 @@ ambient namespaces:
 const rootProgram = 'root'
 const tsTypePrefix = 'type:'
 
+/**
+ * Detect function overloads like:
+ * ```js
+ * export function foo(a: number) {}
+ * export function foo(a: string) {}
```

Comment:
don’t overloads look like:
```suggestion
 * export function foo(a: string);
```
ie, the body is supposed to be omitted?

Reply:
Sure.